tests/test_architecture.py

```diff
@@ -424,18 +424,6 @@ def test_one_hot_encoding_04(self):
         with pytest.raises(ValueError):
             mfe.fit(X=X, y=y, transform_cat="one-hot")
 
-    def test_ohe_full_encoding_missing_value(self):
```

Comment:
Isn't it important to maintain this kind of test?

Reply:
We can maintain this test only if we manually check for NaN values while performing one-hot encoding, since in recent versions of scikit-learn the NaN values does not raises errors while one-hot encoding anymore. Apparently, NaN are now considered a category on its own.

Reply:
I think this behavior is valid. We can follow in this direction. Thank you.